datadog_checks_dev/datadog_checks/dev/tooling/license_headers.py

```diff
@@ -93,8 +100,14 @@ def validate_license_header(path):
         elif license_header != get_default_license_header():
             return LicenseHeaderError("file does not match expected license format", relpath)
 
+    if repo_root:
+        gitignore_matcher = _GitIgnoreMatcher.from_path_to_root(check_path, repo_root)
+    else:
+        gitignore_matcher = _GitIgnoreMatcher(check_path)
```

Comment:
Would it be difficult to include the `git config core.excludesFile` as the root .gitignore's parent as well?

I use one personal .gitignore file and that could be useful.

Reply:
The goal is to try to make the validation work the same locally as it does in CI, and excluding files that the CI would not exclude would work against that goal. Therefore, I don't think it would be particularly useful.

Reply:
Sounds good